libs/facilities/frontend/view/src/lib/components/detail/detail.page.html

```diff
@@ -1,134 +1,157 @@
+                            {{ locked() ? 'Unlock the doors' : 'Lock the doors' }}
                         </ix-button>
                     </div>
-                </div>
-
-            </ix-card-content>
-        </ix-card>
+                </ix-card-content>
+            </ix-card>
 
-        <ix-card variant="insight" class="h-full w-full p-4">
-            <ix-card-content>
-                <ix-card-title>
-                    <ix-icon
-                        class="icon"
-                        [name]="locked() ? 'lock-filled' : 'unlock-filled'"
-                    ></ix-icon>
-                </ix-card-title>
-                <ix-typography color="std" format="h4">
-                    {{ locked() ? 'Unlock the doors' : 'Lock the doors' }}
-                </ix-typography>
-                <ix-typography color="std">
-                    {{ locked() ? 'Currently the doors are locked' :
-                    'Currently the doors are unlocked' }}
-                </ix-typography>
-
-                <div class="mt-auto p-2">
-                    <ix-button (click)="changeLocked()">
-                        {{ locked() ? 'Unlock the doors' : 'Lock the doors' }}
-                    </ix-button>
-                </div>
-            </ix-card-content>
-        </ix-card>
+            <ix-card variant="insight" class="h-full w-full p-4">
+                <ix-card-content>
+                    <ix-card-title>
+                        <ix-icon class="icon" name="earth-filled"></ix-icon>
+                        <span class="notification text-2xl">80%</span>
+                    </ix-card-title>
+                    <ix-typography color="std" format="h4"> Eco Score (SiGreen)</ix-typography>
+                    <ix-typography color="std">
+                        According to SiGreen this facility has an eco score of 80%
+                    </ix-typography>
+                </ix-card-content>
+            </ix-card>
+        </div>
+    </ix-content>
 
-        <ix-card variant="insight" class="h-full w-full p-4">
-            <ix-card-content>
-                <ix-card-title>
-                    <ix-icon class="icon" name="earth-filled"></ix-icon>
-                    <span class="notification text-2xl">80%</span>
-                </ix-card-title>
-                <ix-typography color="std" format="h4"> Eco Score (SiGreen)</ix-typography>
-                <ix-typography color="std">
-                    According to SiGreen this facility has an eco score of 80%
-                </ix-typography>
-            </ix-card-content>
-        </ix-card>
-    </div>
-</ix-content>
+} @else {
+    <ix-content-header
+        class="pl-4 pt-4"
+        hasBackButton="true"
+        headerTitle="Facility Details"
+        headerSubtitle="On this page you can see detailed information about the facility including charts about the pump and environment data"
+        (backButtonClick)="location.back()"
+    ></ix-content-header>
```

Comment:
put the \<ix-content-header\> outside of the if/else again because it is duplicate code